From a976cb05b4024b7a6452d1541378d718cdfe33e6 Mon Sep 17 00:00:00 2001
From: Takuto Ikuta <tikuta@chromium.org>
Date: Thu, 25 Apr 2024 07:25:32 +0000
Subject: [PATCH] [devtools] fix a missing build dependency to a generated file

Bug: 336911498
Change-Id: I6e6d3afaf33ace53a68271b70165b8c3ab596340
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5487538
Auto-Submit: Takuto Ikuta <tikuta@chromium.org>
Commit-Queue: Takuto Ikuta <tikuta@chromium.org>
Reviewed-by: Danil Somsikov <dsv@chromium.org>
Commit-Queue: Danil Somsikov <dsv@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1292300}
---
 chrome/browser/devtools/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/devtools/BUILD.gn b/chrome/browser/devtools/BUILD.gn
index f0c07cad8cfb..053199e462f5 100644
--- a/chrome/browser/devtools/BUILD.gn
+++ b/chrome/browser/devtools/BUILD.gn
@@ -117,6 +117,7 @@ static_library("devtools") {
     "//chrome/browser/autofill:autofill",
     "//components/autofill/content/browser:browser",
     "//components/autofill/core/browser:browser",
+    "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags:buildflags",
     "//components/variations/service:service",
     "//components/webapps/common:common",
